docs/api.rst.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2019 Kattni Rembor for Adafruit Industries
+
+SPDX-License-Identifier: MIT
```

Comment:
Add newline to end of file.

See if you have a setting to do this automatically in your editor. In emacs, if you set removing trailing whitespace, it will ensure there's always a newline at the end of the file.

Reply:
This is not manually edited, but rather created by `reuse addheader` — this is https://github.com/fsfe/reuse-tool/issues/187

But! It turns out that pre-commit has a check/fixer for that ;) So I'll add that to the pull request.